### ISSUE TITLE
Shortcut for Launch Center Pro

### ISSUE THREAD

**Issue by codecypher** [Ranchero-Software/NetNewsWire#3185]
Hi

It would be great if launch center pro could recognize the app and create a shortcut

**Comment by vincode-io** [comment]
I'm not very familiar with Launch Center Pro, but I think the integration would have to be done on their side.

If all you want to do is launch NNW, couldn't you create a launch Shortcut in the Shortcuts app and then import that into Launch Center Pro?

**Comment by codecypher** [comment]
﻿Hi

It’s okay, it was just a feature request. 

Here is an article for app developers describing URL schemes:

http://help.contrast.co/hc/en-us/articles/200865273

I havent looked at the IPA file for your app, but on iOS the URL scheme is part of the Info.plist package config file which is a lot of work for the user to do manually for each app they want to setup:

https://www.amerhukic.com/finding-the-custom-url-scheme-of-an-ios-app

You just fill out a short form to submit the URL scheme for your app so that launch center knows it. I do it for my ios mobile apps since a lot of developers like using launch center. 

FYI shortcuts is not the preferred method for creating app shortcuts, it is very slow and always launches the shortcuts app which is very annoying. 

Jeff Holmes

On Jun 23, 2021, at 1:22 PM, Maurice Parker ***@***.***> wrote:

﻿
I'm not very familiar with Launch Center Pro, but I think the integration would have to be done on their side.

If all you want to do is launch NNW, couldn't you create a launch Shortcut in the Shortcuts app and then import that into Launch Center Pro?

**Comment by vincode-io** [comment]
We've already got the custom URL scheme `netnewswire` set up on iOS.

@brentsimmons Could you fill out this form and submit it? 
https://help.contrast.co/hc/en-us/articles/201402467

I don't mind doing it if you are too busy. I can fill it out with your info if you want so that you are the contact point incase something breaks.

**Comment by codecypher** [comment]
Yes the URL scheme works. Sorry, it’s just a pain trying to find out what they are for each app.

Thanks

Jeff

On Jun 23, 2021, at 4:24 PM, Maurice Parker ***@***.***> wrote:


We've already got the custom URL scheme netnewswire set up on iOS.

@brentsimmons <https://github.com/brentsimmons> Could you fill out this form and submit it?
https://help.contrast.co/hc/en-us/articles/201402467 <https://help.contrast.co/hc/en-us/articles/201402467>
I don't mind doing it if you are too busy. I can fill it out with your info if you want so that you are the contact point incase something breaks.

**Comment by brentsimmons** [comment]
Do we still need a shortcut for Launch Center Pro?

**Comment by codecypher** [comment]
Well, LCP still does not find the NetNewsWire app using the Action Composer. Jeff HolmesOn Mar 8, 2023, at 11:33 AM, Brent Simmons ***@***.***> wrote:﻿
Do we still need a shortcut for Launch Center Pro?
